Documentation/getting-started.md

```diff
@@ -1,89 +1,114 @@
 
 ### Domain configuration
 
 The value under the **Edge Network domain** field is used for requests to Adobe Experience Platform Edge Network and it usually follows the format `<company>.data.adobedc.net`, where `<company>` is the unique namespace associated to your Adobe organization.
 
 If you have a first-party domain mapped to the Adobe-provisioned Edge Network domain, you can enter it here. For more details about how to configure or maintain a first-party domain, see [Adobe-Managed Certificate Program](https://experienceleague.adobe.com/docs/core-services/interface/administration/ec-cookies/cookies-first-party.html?lang=en#adobe-managed-certificate-program).
 
-> **Note**
+> **Note**  
 > The domain name is expected to be just the domain without any protocol or trailing slashes. If no domain is provided, by default the `edge.adobedc.net` domain is used.
 
 ## Add the Edge Network extension to your app
 
-### Download and import the Edge extension
+The Edge Network extension depends on the following extensions:
+* [Mobile Core](https://github.com/adobe/aepsdk-core-android)
+* [Identity for Edge Network](https://github.com/adobe/aepsdk-edgeidentity-android) (peer dependency for the Edge Network extension)
+
+1. Add the Mobile Core, Identity for Edge Network, and Edge Network extensions to your project using the app's Gradle file:
 
+```gradle
+implementation 'com.adobe.marketing.mobile:core:2.+'
+implementation 'com.adobe.marketing.mobile:edge:2.+'
+implementation 'com.adobe.marketing.mobile:edgeidentity:2.+'
+```
+
+> **Warning**  
+> Using dynamic dependency versions is not recommended for production apps. Refer to this [page](https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/MobileCore/gradle-dependencies.md) for managing Gradle dependencies.
+
+2. Import the libraries:
+#### Java
+```java
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Edge;
+import com.adobe.marketing.mobile.edge.identity.Identity;
+```
+
+#### Kotlin
+```kotlin
+import com.adobe.marketing.mobile.MobileCore
+import com.adobe.marketing.mobile.Edge
+import com.adobe.marketing.mobile.edge.identity.Identity
+```
+
+3. Register the Edge extensions with Mobile Core:
+
+#### Java
+```java
+public class MainApp extends Application {
+
+  private final String ENVIRONMENT_FILE_ID = "YOUR_APP_ENVIRONMENT_ID";
+
+	@Override
+	public void onCreate() {
+		super.onCreate();
+
+		MobileCore.setApplication(this);
+		MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID);
+
+		MobileCore.registerExtensions(
+			Arrays.asList(Consent.EXTENSION, Edge.EXTENSION, Identity.EXTENSION),
+			o -> Log.d("MainApp", "Adobe Experience Platform Mobile SDK was initialized.")
+		);
+	}
+}
+```
 
+#### Kotlin
+```kotlin
+class MainApp : Application() {
 
-1. Add the Mobile Core, Identity for Edge Network, and Edge Network extensions to your project using the app's Gradle file:
+  private var ENVIRONMENT_FILE_ID: String = ""
```

Comment:
nit: you can use ENVIRONMENT_FILE_ID: String = "YOUR_APP_ENVIRONMENT_ID" similar to java code